website/blog/bazelcon-2023-recap.mdx

```diff
@@ -31,32 +31,32 @@ More details below!
 
 By Son Luong Ngoc
 
-[![](../static/img/blog/bazelcon-2023-multiplatform.webp)](https://youtu.be/8Dc8R_Zrf6M?t=6148)
+[![](../static/img/blog/bazelcon-2023-multiplatform.webp)](https://www.youtube.com/watch?v=XimUovAh46k&list=PLxNYxgaZ8Rsefrwb_ySGRi_bvQejpO_Tj&index=4&pp=iAQB)
```

Comment:
Maybe since this one is (unfortunately) missing audio, we should put the other two first.

Reply:
sg, done